server/src/test/java/org/opensearch/cluster/routing/remote/RemoteRoutingTableServiceTests.java

```diff
@@ -105,25 +106,28 @@ public void setup() {
             .build();
         clusterSettings = new ClusterSettings(settings, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS);
         clusterService = mock(ClusterService.class);
+        blobStoreTransferService = mock(BlobStoreTransferService.class);
         when(clusterService.getClusterSettings()).thenReturn(clusterSettings);
         blobStoreRepository = mock(BlobStoreRepository.class);
         when(blobStoreRepository.getCompressor()).thenReturn(new DeflateCompressor());
         blobStore = mock(BlobStore.class);
         blobContainer = mock(BlobContainer.class);
-        when(repositoriesService.repository("routing_repository")).thenReturn(blobStoreRepository);
+        when(repositoriesService.repository(anyString())).thenReturn(blobStoreRepository);
```

Comment:
Why was this change required?

Reply:
Did it while the tests were broken, reverted it.